### ISSUE TITLE
Scroll position Build window changes with Ctrl+Alt+L 

### ISSUE THREAD

**Issue by IRooc** [focus-editor/focus#352]
**Describe the bug**
When you have a long output in the build window (like a running program debug log) and you do Ctrl+Alt+L to make the build window bigger the scroll stays at the current bottom, But if you toggle the buildwindow back to smaller version the top scroll position stays the same and the end of the output scrolls out of view

**To Reproduce**
1. Execute a build command that has more than a build window height as output
2. Focus the build window and toggle the large version Ctrl+Alt+L
3. Toggle back to the smaller version Ctrl+Alt+L 
4. See the scroll position stuck to the top of the window instead of the bottom

**Expected behavior**
I would expect the bottom to be the sticky part somehow. But feel free to disagree if your usecase is different.

 - OS: Windows
 - Focus Version: 0.3.4-3

**Additional context**
 Also what to do if there was a manual scroll in the build window already? Or if there is some text selected? Currently if in the big window you select somehing in the bottom of the window, and then toggle it to smaller again the selection is scrolled out of view. But if you change this to stick to the bottom part, the selection on the top part of the big window would scroll out of view...

**Comment by focus-editor** [comment]
I agree it would be nice to have a sticky bottom in the build output window.
